.github/dependabot.yml

```diff
@@ -40,13 +40,8 @@ updates:
   - yguedidi
   - Kdecherf
   ignore:
-  - dependency-name: lcobucci/jwt
-    versions:
-    - ">= 4.2.0"
-  # until we add support for Symfony 5+
   - dependency-name: symfony/*
-    versions:
-    - ">= 5.0.0"
+    update-types: [ "version-update:semver-major" ]
```

Comment:
Updated this way so that the rule can remain across all coming Symfony upgrades, as we want to handle major version upgrade manually